sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs

```diff
@@ -213,7 +213,7 @@ public async Task Batch_SasUri_NotOwner()
             Assert.AreEqual(403, ex.Status);
         }
 
-        [Test]
+       [RecordedTest]
         [LiveOnly]
```

Comment:
I think we shouldn't mark `[LiveOnly]` test with `[RecordedTest]` . I understand `Live` wins here but it looks kind of weird in the code.

Reply:
Since there is no negative side effect of using the attribute in combination with `[LiveOnly]`, leaving it in place seems more flexible if we ever decide to remove `[LiveOnly]`. I'll revert if you feel strongly though.

Reply:
I'd argue that when we touch this test and remove `[LiveOnly]` it's easy enough to add 'Recording' to `Test]`.
Maybe we should have `[LiveOnlyTest]` instead of `[LiveOnly]`.
My main problem is that leaving both in place looks like oxymoron... and might confuse people onboarding to the codebase.
Let's see what others think. @seanmcc-msft @pakrym could you chime in?

Reply:
I'm still not sure I fully understand the value `[RecordedTest]` is adding...I don't fine the current workflow of manually re-recording to be tedious.

Reply:
I liked it while working on new features, one less step in the loop. But we should discuss this in different comment chain if needed.

Reply:
I can see both sides here. Having RecordedTest and LiveOnly does look strange. But I also like it when I have fewer rules to follow so marking everything with `RecordedTest` and using `LiveOnly` is good as well.

Long story short it's up to your team :)

>Maybe we should have [LiveOnlyTest] instead of [LiveOnly].

We can't. `LiveOnly` can be applied to classes and assemblies. And I'm afraid that adding LiveOnlyTest in addition to LiveOnly would only create confusion.